requests_aws4auth/test/requests_aws4auth_test.py

```diff
@@ -182,7 +182,11 @@ def request_from_text(text):
         hdr = hdr.lower()
         vals = headers.setdefault(hdr, [])
         vals.append(val)
-    headers = {hdr: ','.join(sorted(vals)) for hdr, vals in headers.items()}
+
+    headers = {}
```

Comment:
This line doesn't belong here, it empties the headers list and breaks lots of tests.